### ISSUE TITLE
Feel confusion about LoadError when require 'bigdecimal/util'

### ISSUE THREAD

**Issue by tan-wei** [ruby/bigdecimal#137]
OS: Windows 10
Ruby: 2.5.0 x64 with MSYS2
bigdecimal: 1.4.3

When `bigdecimal` is loaded before `bigdecimal/util`:
```ruby
require 'bigdecimal'
require 'bigdecimal/util'
```
It will raise LoadError:
```
uby25-x64/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:130:in `require': 127: ?????????   - C:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.3/lib/bigdecimal/util.so (LoadError)
	from C:/Ruby25-x64/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:130:in `rescue in require'
	from C:/Ruby25-x64/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:34:in `require'
	from C:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.3/lib/bigdecimal/util.rb:9:in `<top (required)>'
	from C:/Ruby25-x64/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from C:/Ruby25-x64/lib/ruby/site_ruby/2.5.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from E:/DemoProjects/RubyLearning/chapter_2.rb:2:in `<main>'
```
However, `bigdecimal/util` is loaded before `bigdecimal` will suceess:
```ruby
require 'bigdecimal/util'
require 'bigdecimal'
```

So why does this happen? It will make library `monetize` or `money` raising `LoadError`, because in their source codes, `bigdecimal` is loaded before `bigdecimal/util`.

**Comment by kou** [comment]
Could you try again with the latest Ruby?

**Comment by tan-wei** [comment]
> 
> 
> Could you try again with the latest Ruby?

Fixed after I install Ruby 2.6.3-x64. Thanks!
